server/src/main/java/io/unitycatalog/server/persist/utils/FileUtils.java

```diff
@@ -32,10 +32,24 @@ private static String getStorageRoot() {
     return properties.getProperty("storageRoot");
   }
 
+  // This is temporary until storageRoot is fully supported
+  private static String getModelStorageRoot() {
+    return properties.getProperty("registeredModelStorageRoot");
```

Comment:
I didn't want to introduce a base storage root since we have nothing that uses it yet aside from models.  Let me know if you prefer that I change this to use the `storageRoot` property now.